doc/layouts/partials/content-header.html

```diff
@@ -0,0 +1,40 @@
+{{ if eq $.Page.Params.design_doc true }}
```

Comment:
May I know how will this page be rendered? Is it a page provided similar as homepage by github with Hugo, in that way, a page will be available under: https://xen-api.github.io/

Reply:
It's a table with metadata about the design like on the old pages:

E.g.: 
![Screenshot 2024-06-04 at 13 58 02](https://github.com/xapi-project/xen-api/assets/413005/f16b3a1f-404a-4ae1-a1a9-dbf8cb86e3e4)